model/config.go

```diff
@@ -128,9 +128,9 @@ const (
 
 	SupportSettingsDefaultTermsOfServiceLink = "https://mattermost.com/terms-of-use/"
 	SupportSettingsDefaultPrivacyPolicyLink  = "https://mattermost.com/privacy-policy/"
-	SupportSettingsDefaultAboutLink          = "https://about.mattermost.com/default-about/"
-	SupportSettingsDefaultHelpLink           = "https://about.mattermost.com/default-help/"
-	SupportSettingsDefaultReportAProblemLink = "https://about.mattermost.com/default-report-a-problem/"
+	SupportSettingsDefaultAboutLink          = "https://docs.mattermost.com/about/product.html/"
```

Comment:
@cwarnermm Like I mentioned in the web app PR, we need to keep these old redirects around since almost all existing Mattermost instances will be pointing to them.

I still think it's a good idea to have them pointing to redirects as well for a similar reason, but that's not as necessary